README.md

```diff
@@ -24,6 +24,7 @@ Currently we support the following monitors:
 - [Uptime](http://uptime.com) ([Additional Config](docs/uptime-configurations.md))
 - [Updown](https://updown.io/)([Additional Config](docs/updown-configuration.md))
 - [Application Insights](https://docs.microsoft.com/en-us/azure/azure-monitor/app/monitor-web-app-availability)([Additional Config](docs/appinsights-configuration.md))
+- [gcloud](https://cloud.google.com/monitoring/uptime-checks) ([Additional Config](docs/gcloud-configuration.md))
```

Comment:
I think we need to add (Not fully tested) because we are skipping its tests on CI

Reply:
For a little note in the `README` mentioning https://github.com/jshmrtn, I'd be happy to provide an account for the CI.

Reply:
I think its a good idea. what do you think? @rasheedamir

Reply:
```patch
diff --git a/README.md b/README.md
index 9a4fa77..ada80e3 100644
--- a/README.md
+++ b/README.md
@@ -116,6 +116,8 @@ or contact us in case of professional services and queries on <hello@stakater.co
 [website]: http://stakater.com/
 [community]: https://www.stakater.com/projects-overview.html
 
+The Google Cloud test infrastructure is sponsored by [JOSHMARTIN])(https://github.com/jshmrtn).
+
 ## Contributers
 
 Stakater Team and the Open Source community! :trophy:
```

Reply:
agreed sure!